searxng_extra/update/update_ahmia_blacklist.py

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env python
+# lint: pylint
 # SPDX-License-Identifier: AGPL-3.0-or-later
+"""This script saves Ahmia's blacklist for onion sites.
 
-# This script saves Ahmia's blacklist for onion sites.
-# More info in https://ahmia.fi/blacklist/
+Output file: :origin:`searx/data/ahmia_blacklist.txt` (:origin:`CI Update data
+...  <.github/workflows/data-update.yml>`).
+
+More info in https://ahmia.fi/blacklist/
```

Comment:
It's a bit nicer if "Ahmia's blacklist" is linked to https://ahmia.fi/blacklist/ instead of mentioning the link afterwards.

Reply:
fixed : https://return42.github.io/searxng/dev/searxng_extra/update.html#update-ahmia-blacklist-py